src/CallHandler.js

```diff
@@ -403,6 +415,8 @@ async function _startCallApp(roomId, type) {
         conferenceId: confId,
         isAudioOnly: type === 'voice',
         domain: jitsiDomain,
+        auth: jitsiAuth,
+        roomId: roomId,
```

Comment:
The room ID doesn't need to be baked into the data - it is accessible with `$matrix_room_id` or because we're doing custom handling we can append it to the false query string.